Datastore/src/DatastoreClient.php

```diff
@@ -530,12 +543,14 @@ public function allocateIds(array $keys, array $options = [])
      *
      *     @type array $transactionOptions Transaction configuration. See
      *           [ReadWrite](https://cloud.google.com/datastore/docs/reference/rest/v1/projects/beginTransaction#ReadWrite).
+     *     @type string $databaseId The value of databaseId to be used for queries.
```

Comment:
```suggestion
     *     @type string $databaseId ID of the database to which the entities belong.
```

The phrasing of the comment in the constructor option reads nicely to me, would you be open to updating the other locations to match as well?